codesandbox/codesandbox-client
# Add Basic auth as option in private npm

There are also cherry picked type fixes in here, will be removed once #5565 is merged.

To test:
1. Set npm registry on "Custom"
2. Set auth on "Basic"
3. Verify that the Authentication header is Basic on the server
